jetty-core/jetty-deploy/src/main/config/etc/jetty-deployment-scanner.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://jetty.org/configure_10_0.dtd">
+
+<!-- =============================================================== -->
+<!-- Create Default Deployment Provider                              -->
+<!--                                                                 -->
+<!-- The Default Deployment Provider is responsible for monitoring   -->
+<!-- provided directories for web applications and hot deploying     -->
+<!-- through the DeploymentManager.                                  -->
+<!-- =============================================================== -->
+<Configure id="deploymentScanner" class="org.eclipse.jetty.deploy.DeploymentScanner">
+  <Arg name="server">
+    <Ref refid="Server" />
+  </Arg>
+  <Arg name="deploymentManager">
```

Comment:
This arg cannot be name `deploymentManager` as it prevents other types of `ContextHandlerDeployers` from being passed in.     The abstract form should be renamed `contextHandlerDeployer` everywhere in the XML and the @Name annotations